theano/sandbox/cuda/tests/test_basic_ops.py

```diff
@@ -945,14 +945,14 @@ class TestAlloc(theano.tensor.tests.test_basic.TestAlloc):
     dtype = "float32"
     mode = mode_with_gpu
     shared = staticmethod(cuda.shared_constructor)
-    allocs = [B.GpuAlloc, B.GpuAlloc, tensor.Alloc]
+    allocs = [B.GpuAlloc(), B.GpuAlloc(), tensor.Alloc()]
 
 
 class T_Join_and_Split(theano.tensor.tests.test_basic.T_Join_and_Split):
     def setUp(self):
         utt.seed_rng()
         self.mode = mode_with_gpu.excluding('constant_folding')
-        self.join_op = cuda.GpuJoin
+        self.join_op = cuda.GpuJoin()
         # No gpu split.
         self.split_op = tensor.Split
```

Comment:
This line is a good example of possible confusion @lamblin talk about. Can we at least make it consistent inside one class? I'm ok to not change the name if it is consistent in one class. Do you know if other class still use the old definition? If yes, we could make a ticket to make this constant.